installer/src/main/java/com/teamscale/profiler/installer/InstallCommand.java

```diff
@@ -0,0 +1,37 @@
+package com.teamscale.profiler.installer;
+
+import okhttp3.HttpUrl;
+import picocli.CommandLine;
+
+import java.util.concurrent.Callable;
+
+import static com.teamscale.profiler.installer.RootCommand.EXIT_CODE_PROGRAMMING_ERROR;
+
+/**
+ * CLI command that installs the profiler.
+ */
+@CommandLine.Command(name = "install", description = "Installs the profiler.")
+public class InstallCommand implements Callable<Integer> {
+
+	@CommandLine.Parameters(index = "0", converter = HttpUrlTypeConverter.class, description = "The URL of your Teamscale instance.")
+	private HttpUrl teamscaleUrl;
+
+	@CommandLine.Parameters(index = "1", description = "The user used to access your Teamscale instance and upload coverage to your projects.")
```

Comment:
tbh during the installation it can be any valid user. We can only check if the user can access Teamscale. Since during installation we don't have any particular project yet that we know should be accessed, we can't check permissions at this point other than "can authenticate with Teamscale".